lesson-6/index.html

```diff
@@ -25,6 +25,9 @@
   </header>
   <section>
    <h1 class="text-discover">Discover stays <br> to live, work or just relax</h1>
+   <div class="place-of-residence">
+     <p>Your destination or hotel name</p><p>Check-in — Check-out</p>
+   </div>
    <div class="search">
      <input type="text" id="newyork" value="New York">
```

Comment:
bad naming of id

Reply:
you don't need to use margin here. To resize the input itself, you should use width and height
![image](https://user-images.githubusercontent.com/20189313/127902674-6b668276-d808-4687-98aa-ebe754b1f934.png)

Reply:
and background-image hasn't worked yet